Tests/Unit/Service/RegistrationManagerTest.php

```diff
@@ -406,6 +407,7 @@ public function testCreatedBookedOutSeminarHasUidGreaterZero()
         );
     }
 
+
```

Comment:
Begone, additional blank line! ;-)

Reply:
This seems to be an issue from GitHub. There is no additional blank line even if you edit the file on GitHub.